<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniMonster Spiel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSpieler1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Neuen Spieler 1
erstellen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineName1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSpieler2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Neuen Spieler 2
erstellen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineName2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKampf">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kampf</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1Mon1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1Mon2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1Mon3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1Nr1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1Nr2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1Nr3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>3.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1HP1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1HP2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1HP3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2HP1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2HP2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2Nr1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2Nr2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2Mon2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2Nr3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>3.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2Mon1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2Mon3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2HP3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>30</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1HPanz1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>40</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1HPanz2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>40</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1HPanz3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>40</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2HPanz3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>40</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2HPanz1">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>40</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2HPanz2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>40</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHeilen1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Monster heilen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHeilen2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Monster heilen</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelKampfName1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelVS">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>vs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelKampfName2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl Monster:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Anzahl Monster:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1MonAnzahl">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2MonAnzahl">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSpieler1speichern">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Spieler speichern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSpieler1laden">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Spieler laden</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFehler">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#e01b24;&quot;&gt;Fehler: Name ist leer oder&lt;br&gt; enthält Leerzeichen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSpieler2speichern">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Spieler speichern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSpieler2laden">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Spieler laden</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFehler2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#e01b24;&quot;&gt;Fehler: Ein Spieler hat&lt;br&gt;keine kampffähigen Monster&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSpieler1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnSpieler1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSpieler2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnSpieler2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSpieler1speichern</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnSpieler1speichern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSpieler1laden</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnSpieler1laden()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSpieler2speichern</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnSpieler2speichern()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSpieler2laden</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnSpieler2laden()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>780</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHeilen2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnHeilen2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHeilen1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnHeilen1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnKampf</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnKampf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnSpieler1()</slot>
  <slot>btnSpieler2()</slot>
  <slot>btnHeilen1()</slot>
  <slot>btnHeilen2()</slot>
  <slot>btnSpieler1speichern()</slot>
  <slot>btnSpieler1laden()</slot>
  <slot>btnSpieler2speichern()</slot>
  <slot>btnSpieler2laden()</slot>
  <slot>btnKampf()</slot>
 </slots>
</ui>
